spring-ai-docs/src/main/antora/modules/ROOT/pages/api/vectordbs/neo4j.adoc

```diff
@@ -15,16 +15,16 @@ link:https://neo4j.com/docs/cypher-manual/current/indexes-for-vector-search/[Neo
 1. OpenAI Account: Create an account at link:https://platform.openai.com/signup[OpenAI Signup] and generate the token at link:https://platform.openai.com/account/api-keys[API Keys].
 
 2. A running Neo4j (5.13+) instance
-a. link:https://hub.docker.com/_/neo4j[Docker] image _neo4j:5.13_
+a. link:https://hub.docker.com/_/neo4j[Docker] image _neo4j:5.15_
 b. link:https://neo4j.com/download/[Neo4j Desktop]
 c. link:https://neo4j.com/cloud/aura-free/[Neo4j Aura]
 d. link:https://neo4j.com/deployment-center/[Neo4j Server] instance
 
 == Configuration
 
-To connect to Neo4j and use the `Neo4jVectorStore`, you need to provide (e.g. via `application.properties`) configurations for your instance.
+To connect to Neo4j and use the `Neo4jVectorStore`, you need to provide (e.g. via environment variables) access details for your instance.
 
-Additionally, you'll need to provide your OpenAI API Key. Set it as an environment variable like so:
+Additionally, you will need to provide your OpenAI API Key. Set it as an environment variable like sos:
```

Comment:
I guess it is should be `so:` instead of `sos:`?

Reply:
Definitely. 😅